web/src/components/PoisMobile.tsx

```diff
@@ -168,7 +173,7 @@ const PoisMobile = ({
         }
       />
       <BottomActionSheet
-        title={!canGoBack ? t('listTitle') : undefined}
+        title={!canGoBack ? '' : undefined}
```

Comment:
This now chooses between two ways that don't show a title, so we can simply remove the entire title.